plugins/linux-v4l2/v4l2-output.c

```diff
@@ -139,18 +160,21 @@ static bool try_connect(void *data, const char *device)
 	struct obs_video_info ovi;
 	obs_get_video_info(&ovi);
 
-	memset(&parm, 0, sizeof(parm));
 	parm.type = V4L2_BUF_TYPE_VIDEO_OUTPUT;
-
 	parm.parm.output.capability = V4L2_CAP_TIMEPERFRAME;
 	parm.parm.output.timeperframe.numerator = ovi.fps_den;
 	parm.parm.output.timeperframe.denominator = ovi.fps_num;
 
 	if (ioctl(vcam->device, VIDIOC_S_PARM, &parm) < 0)
 		goto fail_close_device;
 
+	uint32_t width = obs_output_get_width(vcam->output);
+	uint32_t height = obs_output_get_height(vcam->output);
+	vcam->frame_size = width * height * 2;
+
 	format.fmt.pix.width = width;
 	format.fmt.pix.height = height;
+	format.fmt.pix.field = V4L2_FIELD_NONE;
```

Comment:
Is this line change related to what this PR is about? If it's not related it should probably be a separate PR. Or at least a separate commit.